### DISCUSSION TITLE
Setting fps

### DISCUSSION THREAD

**Discussion by quockhanh1198** [JJFourie/ESP32Cam-MQTT-SPIFFS-PIR#3] [Q&A]
Hi @JJFourie , I read and feel your project is very fantastic, I want to config fps and setting but I don't know to do it, Can you help me, bro ?

**Comment by JJFourie** [comment]
Thanks!
I did not do anything with FPS, so I can't really say anything about it.
Maybe you can learn from what this guy did in the [Espressif github](https://github.com/espressif/esp32-camera/issues/15#issuecomment-455886304), he managed to get 60 FPS in CIF and 30 FPS in VGA/SVGA.

You can also have a look at the paragraph on frame rates in the [OV2640 writeup](https://www.uctronics.com/download/cam_module/OV2640DS.pdf).

**Reply by quockhanh1198** [reply]
Thanks for your reply, but it's important that I want to show the fps api to the browser and be able to change them using the api. For example: http:/ipcam/control?var=fps&val=10 to http: ......../var=fps&val=8.

**Reply by JJFourie** [reply]
You managed to get the FPS sorted?
Else perhaps you could have a look what this guy is doing.
https://github.com/jeanlemotan/esp32-cam-fpv
